client/public/loginPage/loginScript.js

```diff
@@ -0,0 +1,45 @@
+const loginForm = document.getElementById("loginFrm");
+const emailInpt = document.getElementById("emailInpt");
+const passwordInpt = document.getElementById("passwordInpt");
+
+loginForm.addEventListener("submit", async (e) => {
+  e.preventDefault();
+  const userInfo = {
+    email: loginForm.email.value,
+    password: loginForm.password.value,
+  };
+  try {
+    if (userInfo.email !== null || userInfo.password !== null) {
```

Comment:
what if `userInfo.email` doesn't exit? comparing to null can lead to mistakes. 
considier using `if(!userInfo.email || !userInfo.password)`